$NetBSD: patch-src_res_prefs.ad,v 1.1 2025/10/12 07:34:33 vins Exp $

* Add support for XFind in tools menu.
* Change some defaults.

--- src/res/prefs.ad.orig	2025-09-13 08:56:10.000000000 +0000
+++ src/res/prefs.ad
@@ -24,6 +24,14 @@ XFile*toolsMenu.terminal.accelerator: <K
 !! Make sure it remains the first menu item
 XFile*toolsMenu.terminal.positionIndex: 0
 
+!! Add XFind to the Tools menu
+XFile.tools.finder: xfind
+XFile*toolsMenu.finder.labelString: Finder
+XFile*toolsMenu.finder.mnemonic: F
+XFile*toolsMenu.finder.acceleratorText: F11
+XFile*toolsMenu.finder.accelerator: <Key>F11
+XFile*toolsMenu.terminal.positionIndex: 1
+
 !-----------------------------------------------------------------------------
 ! Environment variables used by the file type database
 !-----------------------------------------------------------------------------
@@ -31,9 +39,9 @@ XFile.variable.terminal: xterm -e
 XFile.variable.textEditor: xedit
 XFile.variable.imageViewer: ximaging
 XFile.variable.imageEditor: xpaint
-XFile.variable.audioPlayer: mplayer
-XFile.variable.videoPlayer: mplayer
-XFile.variable.pdfViewer: xpdf
+XFile.variable.audioPlayer: mpv
+XFile.variable.videoPlayer: mpv
+XFile.variable.pdfViewer: mupdf
 XFile.variable.webBrowser: links -g
 
 !-----------------------------------------------------------------------------
